GUIDE_ADD_NEW_DISEASE_VARIANT.md

```diff
@@ -0,0 +1,40 @@
+# How to add a new disease variant?
+
+This guide explains how to add a new disease variant to SORMAS.
+
+## I. Preparation of the data
+
+First step is to retrieve the identifiers of the diseases.
+An exhaustive list can be found inside the enum ```de.symeda.sormas.api.Disease```.
+
+## II. Persist the data
+
+1. Get into pgAdmin or into the postgreSQL shell to be able to execute the following queries into the PostgreSQL database.
+2. Replace the variable ```{{ DISEASE ID }}``` in the following query with the Disease ID taken from the Java enum indicated in step I.
+3. Replace the variable ```{{ VARIANT NAME }}``` in the following query with the name of the disease variant.
+4. Replace the ```[...]``` by all other rows you need to add.
+5. Execute the generated query.
+6. Let's verify that the table ```diseasevariant``` is well completed with your data.
+
+Here is the SQL query to execute:
+
+```sql
+INSERT INTO diseasevariant (id, uuid, creationdate, changedate, disease, name)
+VALUES
+  (nextval('entity_seq'), gen_random_uuid(), now(), now(), '{{ DISEASE ID }}', '{{ VARIANT NAME }}'),
+  [...];
+```
+
+Here is an example of the query with sample data:
+
+```sql
+INSERT INTO diseasevariant (id, uuid, creationdate, changedate, disease, name)
+VALUES
+  (nextval('entity_seq'), gen_random_uuid(), now(), now(), 'YELLOW_FEVER', 'Yellow Fever Variant 1'),
```

Comment:
There is no `gen_random_uuid()` function in a default SORMAS database right now, so this script will not work. If you've created such a function, please include it in `sormas_schema.sql`; otherwise, this script needs to be rewritten

Reply:
This script (and this function) has been tested on the PostgreSQL provided by the docker-compose file in sormas-cargoserver.
This function is provided by the pgcrypto extension that seems to be automatically installed.
In case there's an error when runing this script, I can indicate how to activate this feature in the documentation.

Reply:
Okay, then it just doesn't work on non-Docker versions (because it seems like that extension is missing there); I also expect that the UUIDs provided by that function will not have the same format as the SORMAS UUIDs, but that shouldn't be that big of a problem right now.

Indicating how to activate the feature would indeed be very helpful then :)

Reply:
Documentation modified